### ISSUE TITLE
brctl commands don't stick

### ISSUE THREAD

**Issue by digininja** [botherder/ntap#1]
I've not tested this but aren't the brctl commands only relevant to the current session? After you shut down the bridge will be lost.

Why not add these to the start of your script, you could do a simple check to see if the bridge already exists and has the two interfaces on it before creating it.

I assume that you don't need ip_forward set for this to work.

**Comment by botherder** [comment]
I don't think so, it worked well so far, but yes you're right, would be good to verify the bridge and create it on the fly if needed.

**Comment by tompt** [comment]
hi, i have a issue.
when i reboot PI, it doesnt always detect the second eth.
and ten seconds isnt enough to enter and delete ntap.sh or alter the file.
happens to some dongles i have. any tip?
